i18n/en.json

```diff
@@ -1525,15 +1525,15 @@
   },
   {
     "id": "api.reaction.delete_reaction.mismatched_channel_id.app_error",
-    "translation": "Failed to delete reaction when channel id in URL doesn't match post id in URL"
+    "translation": "Failed to delete reaction when channel ID in URL doesn't match post ID in URL"
```

Comment:
Propose: "Failed to delete reaction because channel ID does not match post ID in the URL."